<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>231</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="serialPortComboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>端口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>51200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>64000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>76800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>153600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>576000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>流控制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openSerialPortButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>CommandLinkButton</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="logTxtBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>241</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>491</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>170</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>170</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>清空(&amp;C)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>150</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>添加换行符\n</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="serialPortSendTextBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>发送Hex</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>发送文本</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>971</width>
      <height>341</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCheckBox" name="authScrollCheckBox">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>310</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>自动滚动</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>增加时间</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hex显示</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>文本显示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>310</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>增加换行</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="serialPortRecvTextBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>941</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="showSNR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>显示信噪比图</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showLocate">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>210</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>显示定位信息图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearRecvDataButton">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>240</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>清空缓冲区</string>
     </property>
    </widget>
    <widget class="QLabel" name="recvBufferSizeLabel">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>250</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveRecvToFileButton">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>270</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
      </font>
     </property>
     <property name="text">
      <string>保存文件</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="configPanelCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>显示定位信息图</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
